ci-operator/step-registry/upi/install/azurestack/upi-install-azurestack-commands.sh

```diff
@@ -0,0 +1,283 @@
+data["data"]["azure_client_secret"] = base64.b64encode(os.environ["AAD_CLIENT_SECRET"].encode("ascii")).decode("ascii");
+data["data"]["azure_tenant_id"] = base64.b64encode(os.environ["TENANT_ID"].encode("ascii")).decode("ascii");
+data["data"]["azure_region"] = base64.b64encode(os.environ["AZURE_REGION"].encode("ascii")).decode("ascii");
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+python3 -c '
+import yaml;
+path = "manifests/cluster-infrastructure-02-config.yml";
+data = yaml.full_load(open(path));
+data["status"]["platformStatus"]["azure"]["cloudName"] = "AzureStackCloud";
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+# typical upi instruction
+python3 -c '
+import yaml;
+path = "manifests/cluster-scheduler-02-config.yml";
+data = yaml.full_load(open(path));
+data["spec"]["mastersSchedulable"] = False;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+# typical upi instruction
+python3 -c '
+import yaml;
+path = "manifests/cluster-dns-02-config.yml";
+data = yaml.full_load(open(path));
+del data["spec"]["publicZone"];
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+azurestackjson=$(cat $SHARED_DIR/azurestackcloud.json)
+
+cat << EOF > openshift/99_openshift-machineconfig_99-master-azurestackcloud.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: null
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-azurestack
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 3.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+        - path: ${SHARED_DIR}/azurestackcloud.json
+          contents:
+            source: data:text/plain;charset=utf-8;base64,$(echo $azurestackjson | base64 | tr -d '\n')
+          mode: 420
+          user:
+            name: root
+    systemd:
+      units:
+        - name: kubelet.service
+          dropins:
+            - name: 10-azurestack.conf
+              contents: |
+                [Service]
+                Environment="AZURE_ENVIRONMENT_FILEPATH=/etc/kubernetes/azurestackcloud.json"
+  fips: false
+  kernelArguments: null
+  kernelType: ""
+  osImageURL: ""
+EOF
+
+INFRA_ID=$(yq r manifests/cluster-infrastructure-02-config.yml 'status.infrastructureName')
+RESOURCE_GROUP=$(yq r manifests/cluster-infrastructure-02-config.yml 'status.platformStatus.azure.resourceGroupName')
+echo "${RESOURCE_GROUP}" > "${SHARED_DIR}/RESOURCE_GROUP_NAME"
+
+openshift-install create ignition-configs &
+
+set +e
+wait "$!"
+ret="$?"
+set -e
+
+cp "${dir}/.openshift_install.log" "${ARTIFACT_DIR}/.openshift_install.log"
+
+if [ $ret -ne 0 ]; then
+  exit "$ret"
+fi
+
+cp -t "${SHARED_DIR}" \
+    "${dir}/auth/kubeadmin-password" \
+    "${dir}/auth/kubeconfig" \
+    "${dir}/metadata.json" \
+    ${dir}/*.ign
```

Comment:
do we need ignition files in the shared dir?

Reply:
I dont think so. Removing them.